pkg/combustion/rpm.go

```diff
@@ -23,26 +24,31 @@ const (
 var modifyRPMScript string
 
 func configureRPMs(ctx *image.Context) ([]string, error) {
-	if !isComponentConfigured(ctx, userRPMsDir) {
+	if SkipRPMComponent(ctx) {
 		log.AuditComponentSkipped(rpmComponentName)
+		zap.L().Info("Skipping RPM component. Configuration is not provided")
 		return nil, nil
 	}
 
-	rpmSourceDir := generateComponentPath(ctx, userRPMsDir)
+	zap.L().Info("Configuring RPM component...")
 
-	rpmFileNames, err := getRPMFileNames(rpmSourceDir)
+	var rpmDir string
+	if isComponentConfigured(ctx, userRPMsDir) {
+		rpmDir = generateComponentPath(ctx, userRPMsDir)
+	}
+
+	repoPath, packages, err := ctx.RPMResolver.Resolve(&ctx.ImageDefinition.OperatingSystem.Packages, rpmDir, ctx.CombustionDir)
```

Comment:
Can we add in some logging output here so it doesn't look like EIB is hanging whilst this is running?

```suggestion
	log.Audit("Resolving package dependencies...")
	repoPath, packages, err := ctx.RPMResolver.Resolve(&ctx.ImageDefinition.OperatingSystem.Packages, rpmDir, ctx.CombustionDir)
```

Which results in:
```
rdo@woden:~/work/eib-test% sudo podman run --rm --privileged -it -v /home/rdo/work/eib-test/:/eib eib:dev-rpm /bin/eib -config-file eib-config-iso.yaml -config-dir /eib -build-dir /eib/_build
Generating image customization components...
Identifier ................... [SUCCESS]
Custom Files ................. [SKIPPED]
Network ...................... [SKIPPED]
Users ........................ [SUCCESS]
Resolving package dependencies...
```

Reply:
Yep, log has been added!